dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -68,4 +70,18 @@ def check_for_unit_completion
       update!(status: COMPLETED)
     end
   end
+
+  private
+  def enroll_in_module(learning_module)
+    return unless learning_module.plc_course_unit == plc_course_unit
```

Comment:
It's also called in line 59 - which takes a list of learning modules that actually come in from an external call. It's meant to guard against someone submitting an arbitrary list of modules